### PR TITLE
RATEST-78 Allergies automated tests

### PR DESCRIPTION
**Pull request by achilep**
Issue I on: https://issues.openmrs.org/browse/RATEST-78

### CODE REVIEW

**Comment by HerbertYiga** [comment]
as you continue working on this, you can also fix the conflicting files!

**Comment by achilep** [comment]
> as you continue working on this, you can also fix the conflicting files!

Thanks @HerbertYiga. cc @kaweesi

File: ui-tests/src/test/java/org/openmrs/reference/page/AllergyPage.java [hunk 1]
```diff
@@ -25,6 +26,15 @@
     private static final By ADD_NO_KNOWN = By
             .xpath("//*[contains(text(),'No Known Allergy')]");
     private static final By DELETE_NO_KNOWN = By.className("delete-action");
+    public static final By CHOOSEDRUG = By.id("allergen-162299");
+    public static final By CHOOSEREACTION = By.id("reaction-121629");
```
Comment (HerbertYiga) [inline ui-tests/src/test/java/org/openmrs/reference/page/AllergyPage.java:30]: are you able to reproduce  these ids (allergen-162299, reaction-121629) both locally and remotely on running openmrs instances?

File: ui-tests/src/test/java/org/openmrs/reference/page/AllergyPage.java [hunk 2]
```diff
@@ -68,6 +78,68 @@ public void removeNoKnownAllergy() {
...
+    public void clickOnEdditAllergy(){
+        driver.findElement(EDIT_ALLERGY).click();
+    }
+
+    public void selectAllergyType(int allergytype){
+        driver.findElement(ALLERGYTYPE).getTagName("label")[allergytype].click();
+    }
+
+    public String getSeclectedAllergernclass(int allergytype){
+        WebElement l = driver.findElement(AllERGEN).getTagName("ul")[allergytype];
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/AllergyPage.java:118]: Its a good recommendation to use meaningful variable name in your code for easy code readability

File: ui-tests/src/test/java/org/openmrs/reference/page/AllergyPage.java [hunk 3]
```diff
@@ -68,6 +78,68 @@ public void removeNoKnownAllergy() {
...
+
+    public void selectalleregn(int allergytype){
+        WebElement l = driver.findElement(AllERGEN).getTagName("ul")[allergytype];
+        int allergen = (int)(Math.random()*(3) + 0);
+        l.getTagName('li')[allergen];
+        l.isSelected();
+    }
+
+    public void selectReaction(){
+        WebElement l = driver.findElement(REACTION).getTagName("li")[5];
```
Comment (kdaud) [inline ui-tests/src/test/java/org/openmrs/reference/page/AllergyPage.java:130]: the same here as well in reference to my above comment

**Comment by kdaud** [comment]
@achilep are you still working on this PR ?

**Comment by kdaud** [comment]
Closing this PR for now, it can be be opened if desirable!

**Pull request closed without merging**